project-templates/staff/leagues/league_create.html

```diff
@@ -0,0 +1,161 @@
+                <div class="alert alert-danger">
+                    <strong>{{ error|escape }}</strong>
+                </div>
+            {% endfor %}
+        {% endfor %}
+        {% for error in form.non_field_errors %}
+            <div class="alert alert-danger">
+                <strong>{{ error|escape }}</strong>
+            </div>
+        {% endfor %}
+    {% endif %}
+    <h1><a href="{% url 'staff:list_league' %}">Return to league list</a></h1>
+    <br>
+    <a href="{% url 'staff:tournamentrulesetcreate' %}"><i class="fa fa-plus"></i> Create a new ruleset</a>
+
+    <form enctype="multipart/form-data" method="post">
+        {% csrf_token %}
+        {{ field.errors }}
+
+        <label>Name</label>
+        {{ form.name }}
+        <br>
+
+        <label>Settings</label>
+        {{ form.settings }}
+        <br>
+
+        <label>Team Format</label>
+        {{ form.teamformat }}
+        <br>
+
+        <label>Best of</label>
+        {{ form.bestof }}
+        <br>
+
+
+        <div class="form-group">
+            <div class="input-group">
+                <label>Registration Open(IN UTC)</label>
+                {{ form.open_register }}
+            </div>
+        </div>
+
+        <div class="form-group">
+            <div class="input-group">
+                <label>Registration Close(IN UTC)</label>
+                {{ form.close_register }}
+            </div>
+        </div>
+
+        <label>Required Credits</label>
+        {{ form.req_credits }}
+        <br>
+
+        <label>Max Number of Teams (in the entire league)</label>
+        {{ form.size }}
+        <br />
+
+        {% if ESPORTS_MODE %}
+        <label>Platform</label>
+        {{ form.platform }}
+        <br>
+
+        <label>Game</label>
+        {{ form.game }}
+        <br>
+        {% endif %}
+
+        {% if not ESPORTS_MODE %}
+        <label>Sport</label>
+        {{ form.sport }}
+        <br>
+        {% endif %}
+
+        {% if ESPORTS_MODE %}
+        <label>Map Pool</label>
+        {{ form.maps }}
+        <br>
+        {% endif %}
+
+        <div class="form-group">
+            <div class="input-group">
+                <label>Start Time</label>
+                {{ form.start }}
+            </div>
+        </div>
+        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+
+        <label>Image</label>
+        {{ form.image }}
+        <br>
+
+        <label>Disable User Match Report</label>
+        {{ form.disable_userreport }}
+        <br>
+
+        <label>General Info</label>
+        {{ form.info }}
+        <br>
+
+        <label>Prize 1</label>
+         form.prize1 }}
+        <br>
+
+        <label>Prize 2</label>
+         form.prize2 }}
+        <br>
+
+        <label>Prize 3</label>
+         form.prize3 }}
+        <br>
+
+        <label>Active</label>
+        {{ form.active }}
+        <br>
+
+        <label>Ruleset</label>
+        {{ form.ruleset }}
```

Comment:
Maybe this should be part of league settings?

Reply:
I don't think so, settings are more for a set game and adding this quick field has minimal effect.